_data/animal-crossing-wild-world.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
same as above, this file should be in under folder `scores`

Reply:
I tried but it didn't work. The json file can only be read when it is in _data